nca/Parsers/IngressPolicyYamlParser.py

```diff
@@ -242,10 +243,14 @@ def parse_policy(self):
 
         self.default_backend_peers, self.default_backend_ports = self.parse_backend(policy_spec.get('defaultBackend'),
                                                                                     True)
-        # TODO extend to other ingress controllers
-        res_policy.selected_peers = \
-            self.peer_container.get_pods_with_service_name_containing_given_string('ingress-nginx')
+        ingress_controllers = ['ingress-nginx', 'ingress-gce', 'app-ingress']
```

Comment:
Are the controllers you added also implemented by some service name containing those names?

Reply:
After a short discussion,  we will focus on ingress-nginx at this point.